src/main/java/org/mongen/core/service/CollaboratorService.java

```diff
@@ -41,6 +42,13 @@ public Collaborator createCollaborator(CollaboratorPayload collaborator_payload)
 		Collaborator new_collaborator = new Collaborator(collaborator_payload.getFirst_name(), collaborator_payload.getLast_name(), type, country);
 		return collaboratorRepo.save(new_collaborator);
 	}
+
+	public Collaborator createMainContact(MainContactPayload main_contact_payload) {
```

Comment:
you will have to explain me how this whole thing is working 😱

Reply:
i mean this db querying ini java

Reply:
For sure, will schedule something...

![tenor-84112495](https://user-images.githubusercontent.com/14996980/104715006-d9856c00-571d-11eb-9af1-9cfe062a3859.gif)

Nah, I'm happy to explain it, it's easier than it looks 😄